util/start_test

```diff
@@ -90,7 +90,7 @@ def run_tests(tests):
 
     # print out flags
     logger.write('[tests: "{0}"]'.format(" ".join(files)))
-    if args.recurse:
+    if args.recurse and not os.path.isfile("NORECURSE"):
```

Comment:
How is this feature different from the `NOTEST` file? Could `NOTEST` suffice for your case?